src/nest.js

```diff
@@ -41,7 +43,12 @@ export default function() {
     if (++depth > keys.length) return map;
     var array, sortKey = sortKeys[depth - 1];
     if (rollup != null && depth >= keys.length) array = map.entries();
-    else array = [], map.each(function(v, k) { array.push({key: k, values: entries(v, depth)}); });
+    else {
+      array = [];
+      map.each(function (v, k) {
+        array.push({ [keyField]: k, [valuesField]: entries(v, depth) });
```

Comment:
AFIAK The rest of the D3 codebase is strictly ES5 (except modules which are handled by Rollup). The introduction of this ES6 syntax here may break some environments that expect pure ES5 and are not set up to transpile.

Suggestion: use ES5:

```js
const e = {};
e[keyField] = k;
e[valuesField] = entries(v, depth);
array.push(e);
```

Reply:
@curran ugh :(   I think there should either be a test for that, or it might be time to switch to ES6+, and use babel.  In any case, fixed, thx for spotting.